examples/language-modeling/README.md

```diff
@@ -45,19 +45,57 @@ slightly slower (over-fitting takes more epochs).
 
 We use the `--mlm` flag so that the script may change its loss function.
 
+If use whole-word masking, use both `--mlm` and `--wwm` flag(for English Model).
+
+```bash
+export TRAIN_FILE=/path/to/dataset/wiki.train.raw
+export TEST_FILE=/path/to/dataset/wiki.test.raw
+
+python run_language_modeling.py \
+    --output_dir=output \
+    --model_type=roberta \
+    --model_name_or_path=roberta-base \
+    --do_train \
+    --train_data_file=$TRAIN_FILE \
+    --do_eval \
+    --eval_data_file=$TEST_FILE \
+    --mlm \
+    --wwm
+```
+
+For Chinese Model, we need to generate ref files, case it's char level.
```

Comment:
```suggestion
For Chinese models, we need to generate a reference files, case it's char level.
```
I don't understand the end of the comment: "case it's char level." Could you explain a bit more?